src/org/zaproxy/zap/extension/exportreport/Utility/DualListBox.java

```diff
@@ -0,0 +1,437 @@
+/*
+ * Zed Attack Proxy (ZAP) and its related class files.
+ * 
+ * ZAP is an HTTP/HTTPS proxy for assessing web application security.
+ * 
+ * Licensed under the Apache License, Version 2.0 (the "License"); 
+ * you may not use this file except in compliance with the License. 
+ * You may obtain a copy of the License at 
+ * 
+ *   http://www.apache.org/licenses/LICENSE-2.0 
+ *   
+ * Unless required by applicable law or agreed to in writing, software 
+ * distributed under the License is distributed on an "AS IS" BASIS, 
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied. 
+ * See the License for the specific language governing permissions and 
+ * limitations under the License. 
+ * This file is based on the Paros code file ReportLastScan.java
+ */
+package org.zaproxy.zap.extension.exportreport.Utility;
+
+import java.awt.Color;
+import java.awt.Insets;
+import java.awt.GridBagLayout;
+import java.awt.GridBagConstraints;
+import java.awt.event.ActionEvent;
+import java.awt.event.ActionListener;
+
+import java.util.Arrays;
+import java.util.ArrayList;
+import java.util.Collection;
+import java.util.Iterator;
+import java.util.SortedSet;
+import java.util.TreeSet;
+
+import javax.swing.ListCellRenderer;
+import javax.swing.AbstractListModel;
+import javax.swing.ListModel;
+import javax.swing.JList;
+import javax.swing.JLabel;
+import javax.swing.JTextField;
+import javax.swing.JButton;
+import javax.swing.JPanel;
+import javax.swing.JScrollPane;
+import javax.swing.BorderFactory;
+
+import javax.swing.event.ListSelectionEvent;
+import javax.swing.event.ListSelectionListener;
+
+import org.parosproxy.paros.Constant;
+
+/*
+ * AUTHOR		:	GORAN SARENKAPA - JordanGS
+ * SPONSOR		:	RYERSON UNIVERSITY
+ * REFERENCE	:	http://www.java2s.com/Tutorial/Java/0240__Swing/DualListBoxSample.htm
+ */
+
+public class DualListBox extends JPanel
```

Comment:
Guess the class should be generic (there are a some raw types, also SortedListModel).